src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AttendanceStatistics.kt

```diff
@@ -7,7 +7,7 @@ import uk.gov.justice.digital.hmpps.whereabouts.repository.AttendanceRepository
 import java.time.LocalDate
 
 data class PaidReasons(val attended: Int? = 0, val acceptableAbsence: Int? = 0, val approvedCourse: Int? = 0, val notRequired: Int? = 0)
-data class UnpaidReasons(val refused: Int? = 0, val restDay: Int? = 0, val sessionCancelled: Int? = 0, val sick: Int? = 0, val unacceptableAbsence: Int? = 0, val restInCell: Int? = 0)
+data class UnpaidReasons(val refused: Int? = 0, val restDay: Int? = 0, val sessionCancelled: Int? = 0, val sick: Int? = 0, val unacceptableAbsence: Int? = 0, val restInCell: Int? = 0, val RestInCellOrSick: Int? = 0)
```

Comment:
Are we keeping sick and rest in cell as separate?

Reply:
Going to remove Sick and RestInCell in another PR, along with a migration script to change all historical reasons.